### ISSUE TITLE
decimal rates are bad for business

### ISSUE THREAD

**Issue by catdad** [catdad/grandma#79]
```bash
>grandma run req -d 2m -r 1.12345633464574 --out silly.log
134
{ numerator: 224691267, denominator: 200000000000 }
```

**Comment by catdad** [comment]
Published in 0.4.0.

**Comment by catdad** [comment]
Here are some examples of rate adjustment:

| Input | Real value used |
| --- | --- |
| `1.1166666666666667` or `67/60` | `1.278772378516624` |
| `1.12345633464574` | `1.2853470437017993` |
| `2.56` | `2.5641025641025643` |
| `3.14` | `3.205128205128205` |

**Comment by catdad** [comment]
| Input | Real value used |
| --- | --- |
| `1.1166666666666667` or `67/60` | `1.2165450121654502` |
| `1.12345633464574`                      | `1.2224938875305624` |
| `2.56`                                             | `2.5641025641025643` |
| `3.14`                                             | `3.205128205128205` |

**Comment by catdad** [comment]
Update published in 0.4.1.
